### DISCUSSION TITLE
hudi 0.8.0与flink 1.11代码集成问题

### DISCUSSION THREAD

**Discussion by Liuwenli11** [MyLanPangzi/flink-demo#4] [Q&A]
已解决

**Comment by MyLanPangzi** [comment]
怎么不用master分支,0.8的用的旧的table sink兼容不太好.

**Comment by Liuwenli11** [comment]
兼容不太好的意思除了兼容性之外有性能或者功能问题吗。想线上用，怕master分支没有发版不稳定，以后升级版本的话应该比较容易吧？


 



 方便加微信吗？


 微信是18810698681
 
 
 
 
 
 
 

  -- 
 
 
 发自新浪邮箱客户端 
 
 
  
  在 5月21日 22:03，hiscat ***@***.***> 写道： 
   
  怎么不用master分支,0.8的用的旧的table sink兼容不太好.
